commands/hubdb/delete.ts

```diff
@@ -4,34 +4,61 @@ const { logError } = require('../../lib/errorHandlers/index');
 const { deleteTable } = require('@hubspot/local-dev-lib/api/hubdb');
 const { loadAndValidateOptions } = require('../../lib/validation');
 const { trackCommandUsage } = require('../../lib/usageTracking');
-
 const {
   addConfigOptions,
   addAccountOptions,
   addUseEnvironmentOptions,
 } = require('../../lib/commonOpts');
+const {
+  selectHubDBTablePrompt,
+} = require('../../lib/prompts/selectHubDBTablePrompt');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
 const { i18n } = require('../../lib/lang');
 
 const i18nKey = 'commands.hubdb.subcommands.delete';
 
-exports.command = 'delete <tableId>';
+exports.command = 'delete [table-id]';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.handler = async options => {
-  const { tableId, derivedAccountId } = options;
+  const { force, derivedAccountId } = options;
 
   await loadAndValidateOptions(options);
 
   trackCommandUsage('hubdb-delete', null, derivedAccountId);
 
   try {
+    const { tableId } =
+      'tableId' in options
+        ? options
+        : await selectHubDBTablePrompt({
+            accountId: derivedAccountId,
+            options,
+            isDeleteCommand: true,
+          });
+
+    if (!force) {
+      const { shouldDeleteTable } = await promptUser({
+        name: 'shouldDeleteTable',
+        type: 'confirm',
+        message: i18n(`${i18nKey}.shouldDeleteTable`, { tableId }),
+      });
+      process.stdin.resume();
+
+      if (!shouldDeleteTable) {
+        process.exit(EXIT_CODES.SUCCESS);
```

Comment:
Should we add a log stating we've canceled the delete?

Reply:
I think it's clear, because we exit immediately after the prompt, Are you sure you want to delete this table? And this is a pattern we use with the `migrate-app` command. Going to resolve for now